### DISCUSSION TITLE
[Win11] taskbar along the top edge of the screen?

### DISCUSSION THREAD

**Discussion by bblackmoor** [Open-Shell/Open-Shell-Menu#1687] [General]
I recently migrated to Windows 11. Is there a way to align the taskbar (or taskbars) to the top edge of the screen (or screens) in OpenShell 4.4.191?

(I know that Windows 11 is a moving target. This is a question, not a complaint.)

**Comment by ge0rdi** [comment]
`Open-Shell` doesn't change taskbar position.

For Windows 11 you can try `ExplorerPatcher`.
It used to allow change of taskbar position.
